bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -1,166 +1,438 @@
+            </h2>
+          </span>
+              </span>
+      </div>
+      
+      
+      {% endif %}
+      {% endblock %}
+      {% block platform_switch %}
+      {% endblock %}
+      {% block notes_heading_secondary %}
     </header>
-  {% endblock %}
-
-    <div class="main-column">
+    {% endblock %}
+    <div class="notes-column">
       {% if new_features %}
       <section class="notes-section" id="new-features">
-        <h3>{{ _('What’s New') }}</h3>
-        <ul class="section-items tagged">
-          {% for note in new_features %}
-          <li {% if not note.tag %}class="untagged"{% endif %} id="note-{{ note.id }}">
-            {% if note.tag %}
-              <b class="tag tag-{{ note.tag.lower() }}">{{ note.tag }}</b>
+        <div class="section-wrapper">
+          {% for note in new_features if note.tag =="New" %}
+          {% if loop.first %}
+          <span class="label_wrapper">
+            <p class="note_label">
+              &nbsp;
+              <i class="fa fa-star">
+              </i>
+              {{ _('new') }} 
+            </p>
+          </span>
+          {% endif %}
+          <span class="note_wrapper">
+            <ul class="section-items tagged">
+              <li {% if not note.tag %}class="untagged"{% endif %} {% if loop.index % 2 != 0 %}class = "gray_note" {% endif %} {% if loop.index % 2 == 0 %}class = "blue_note" {% endif %} id="note-{{ note.id }}">
+                
+                <div class="note_wrap">
+                  {{ note.note|markdown|safe }} 
+                </div>
+              </li>
+            </ul>
+            {% if loop.last %}
+            <br>
+            <br>
+            <br>
             {% endif %}
-              {{ note.note|markdown|safe }}
-          </li>
+          </span>
           {% endfor %}
-        </ul>
-      </section>
-      {% endif %}
-      {% if known_issues %}
-      <section class="notes-section" id="known-issues">
-        <h3>{{ _('Known Issues') }}</h3>
-        <ul class="section-items tagged">
-          {% for note in known_issues %}
-            <li id="note-{{ note.id }}">
-                <b class="tag tag-unresolved">{{ _('unresolved') }}</b>
-                {{ note.note|markdown|safe }}
-            {% if note.fixed_in_release %}
-              <p class="note">
-                <a href="{{ releasenotes_url(note.fixed_in_release) }}">
-                  {{ _('Resolved in v{version_number}')|f(version_number=note.fixed_in_release.version) }}
-                </a>
-              </p>
+        </div>
+        <div class="section-wrapper">
+          {% for note in new_features if note.tag =="Fixed" %}
+          {% if loop.first %}
+          <span class="label_wrapper">
+            <p class="note_label">
+              &nbsp;
+              <i class="fa fa-check">
+              </i>
```

Comment:
It's not great to use `&nbsp;`'s for spacing. It's better to use CSS. Along those lines, it would be better to implement the icon in CSS, too. There's a good example of how to do this [on the share widget](https://github.com/mozilla/bedrock/blob/master/media/css/base/mozilla-share-cta.less#L51), which can be seen in action on https://www.mozilla.org/firefox/hello/ (among other pages).